variables.tf

```diff
@@ -44,3 +49,10 @@ variable "security_group_ids" {
   description = "The security group ids which can be given to the bastion instance, defaults to empty"
   default     = []
 }
+
+variable "tags" {
+  type        = list(any)
```

Comment:
We actually avoid using the `any` type as much as possible. For example, in this particular case, this value can be a list of maps.

Reply:
`list(map(string))` maybe?